src/SIL.XForge.Scripture/Services/NotesFormatter.cs

```diff
@@ -178,7 +178,7 @@ private static void ParseComment(XElement commentElem, Comment comment)
         private static void ParseContents(XElement contentElem, Comment comment)
```

Comment:
This may be a good situation to express and show the behaviour with a unit test. ?

Reply:
I have created a set of unit tests for `NotesFormatter` as I realized there were none really, and the behavior of the class is not crystal clear. I also took the opportunity to document it a bit better, and to clean up various formatting suggestions suggested by Visual Studio.

Reply:
Thank you!